### DISCUSSION TITLE
How can I not show private contributions?

### DISCUSSION THREAD

**Discussion by Ombucha** [lowlighter/metrics#1053] [Q&A] [answered]
Can I get some help?

**Comment by lowlighter** [comment, marked as answer]
Hi :wave: !

This is handled in your GitHub settings accounts for contributions:
![](https://github.com/lowlighter/metrics/blob/master/.github/readme/imgs/setup_private_contributions.dark.png?raw=true)

And if you're talking about private repositories and other related stuff, juste provide a token that doesn't have full `repo` scope.

Hope it helps :slightly_smiling_face:
